atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/metrics/TargetedSweepMetrics.java

```diff
@@ -137,6 +137,10 @@ public void registerEntriesReadInBatch(ShardAndStrategy shardStrategy, long batc
         updateMetricsIfPresent(shardStrategy, metrics -> metrics.registerEntriesReadInBatch(batchSize));
     }
 
+    public void updateSweepDelay(ShardAndStrategy shardStrategy, long delay) {
+        updateMetricsIfPresent(shardStrategy, metrics -> metrics.updateSweepDelay(delay));
+    }
```

Comment:
If I understand correctly, this _just_ updates a metric? Can you rename to `updateSweepDelayMetric` to make sure it is unambiguous?

Also, will this be an expensive metric? Just making sure we're not going back to our old $$$ ways.